src/components/Kv/KvLightbox.vue

```diff
@@ -10,8 +10,8 @@
 			tabindex="500"
 		>
 			<div class="kv-lightbox">
-				<div class="row" id="lightbox-row">
-					<div class="columns" id="lightbox-columns">
+				<div class="row lightbox-row">
+					<div class="columns lightbox-columns">
```

Comment:
@davidskiva Just a heads up incase you use the id lightbox-row or lightbox-columns in your tests. They will need to be changed to a class.